### ISSUE TITLE
Training suddenly stops after some EPOCHs

### ISSUE THREAD

**Issue by arianaa30** [harvitronix/five-video-classification-methods#138]
I was using your lstm training code (the one with highest accuracy on the dataset...I guess #4?). I don't know why the training suddenly stops after the end of some EPOCHs with no errors being shown. The EPOCH's number varies randomly. Once 15, once 22, 27, ..

Is there any reasons why this occurs?!
I'm using the subset of your code prepared here: https://github.com/SBoyNumber1/LSTM-video-classification
